.github/workflows/ci.yml

```diff
@@ -96,8 +96,9 @@ jobs:
                 python-version: ${{ matrix.python-version }}
 
         -   name: Install python dependencies
-            run:
+            run: |
                 pip install -e .[tests]
+                pip install --upgrade git+https://github.com/unkcpz/nest_asyncio.git@master
```

Comment:
Yes, after discussion with Martin, we are unanimous that there is a bug https://github.com/erdewit/nest_asyncio/pull/28  there about setting event loop back after exit from the nested loop. Otherwise, some loop will not stop and keep running which prevent us from starting a new event loop.  However, the maintainers of `nest_asyncio` are not very active, still wait them to look into it.